react/features/base/participants/middleware.js

```diff
@@ -32,9 +32,11 @@ MiddlewareRegistry.register(store => next => action => {
         store.dispatch(localParticipantIdChanged(action.conference.myUserId()));
         break;
 
-    case CONFERENCE_LEFT:
+    case CONFERENCE_LEFT: {
         store.dispatch(localParticipantIdChanged(LOCAL_PARTICIPANT_DEFAULT_ID));
+        store.dispatch(pinParticipant(null));
```

Comment:
I'm afraid I don't understand the offered solution. I think clearing the pinned redux state will be enough.

Reply:
I'll take another look.